device/samples/edge_downstream_client.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import time
+
```

Comment:
hate to block on this but the file is missing the standard microsoft copyright notice:
# Copyright (c) Microsoft. All rights reserved.
# Licensed under the MIT license. See LICENSE file in the project root for
# full license information.